<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagement</class>
 <widget class="QWidget" name="UserManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User Management</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeft</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="class" stdset="0">
      <string>page-title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="formGroupBox">
     <property name="title">
      <string>User Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="usernameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="roleLabel">
        <property name="text">
         <string>Role:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="roleCombo">
        <item>
         <property name="text">
          <string>user</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>admin</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add User</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: #2ecc71;
             color: white;
             border: none;
             padding: 5px 15px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #27ae60;
         }
        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Update User</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: #f1c40f;
             color: white;
             border: none;
             padding: 5px 15px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #f39c12;
         }
        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete User</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: #e74c3c;
             color: white;
             border: none;
             padding: 5px 15px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #c0392b;
         }
        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear Form</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: #95a5a6;
             color: white;
             border: none;
             padding: 5px 15px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #7f8c8d;
         }
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="userTable">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Username</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Role</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
